guides/models/adding_models.md

```diff
@@ -0,0 +1,70 @@
+ # Adding a model
+
+`jiant` supports or can easily be exteneded to support Hugging Face Hugging Face's [Transformer models](https://huggingface.co/transformers/viewer/) since `jiant` utilizes [Auto Classes](https://huggingface.co/transformers/model_doc/auto.html) to determine the architecture of the model used based on the name of the [pretrained model](https://huggingface.co/models). To add a model not currently supported in `jiant`, follow the following steps:
```

Comment:
Typos: "exteneded", "Hugging Face Hugging Face's"

Maybe add a clarifying explanation (and let me know if this is wrong): We do use AutoModels to resolve the model in jiant, but in order to ensure the jiant pipeline works correctly (e.g. matching the correct tokenizer) and to deal with some subtle differences between models, jiant needs to know some additional information/supports specific handling for specific models, so some additional steps are needed to set up a new model from HF in jiant.